### ISSUE TITLE
Contact phone and email address

### ISSUE THREAD

**Issue by jmody** [pinkynrg/convergence2.0#103]
Customer contact information (Phone number and email address) is not visible in ticket header. I do not feel it is user friendly to click another tab in header to get those information. This is important information and it easily accessible in old convergence.

**Comment by pinkynrg** [comment]
@AlessandroMelzi, @e80tom, @raulpm8, @e80sam: do you feel the same way? Is that an information we actually need in the mail header of the ticket? since I'm asking, do you feel the header of the ticket is ok now (beside what requested above), or we need to add/remove info?

**Comment by e80tom** [comment]
Yes, it would be easier to access to put it on the main page. The contact info at least.

**Comment by pinkynrg** [comment]
It would also be nice to have the assignee in the ticket header as well
